js/script.js

```diff
@@ -0,0 +1,48 @@
+$(document).ready(function() {
+
+var searchResults = [];
+var imageArray = [];
+var testImage;
+var counter = 0;
+var searchTerm;
+
+//Get search term from user form
+$('#search-form').submit(function(e) {
+	e.preventDefault();
+	searchTerm = $('#search-term').val();
+
+//AJAX request to Reddit
+$.get('https://www.reddit.com/search.json', {
+	q: searchTerm
+	}).done(function(data) {
+	searchResults = data
+		// console.log(searchResults.data.children);
+	
+//Filter search results to exclude those without images
+	searchResults.data.children.filter(function (child) {
+		if (child.data.preview) {
+			imageArray.push(child.data.preview.images[0].source.url);
+			// console.log(imageArray);
+		}
+		else {
```

Comment:
Also, let's not alert the user here, or else they will get an alert each time filter iterates over the array when there are no pictures for that item! ```console.log``` would probably be the best choice here.